--- PodParser.pm	2013-04-09 18:19:56.000000000 +0100
+++ PodParser.pm	2013-04-09 18:22:14.000000000 +0100
@@ -32,6 +32,14 @@
     last if ($self->{abstract}) = /^ (?: [a-z0-9:]+ | [BCIF] < [a-z0-9:]+ > ) \s+ - \s+ (.*\S) /ix;
   }
 
+  my @description;
+  while (<$fh>) {
+    next unless /^=head1\s+DESCRIPTION/i ... /^=/;
+    push @description, $_ unless /^=/;
+    last if /^=/ and @description;
+  }
+  $self->{description} = \@description;
+
   my @author;
   while (<$fh>) {
     next unless /^=head1\s+AUTHORS?/i ... /^=/;
@@ -63,3 +71,12 @@
 
   return $self->{author} || [];
 }
+
+sub get_description {
+  my $self = shift;
+  return $self->{description} if defined $self->{description};
+
+  $self->parse_from_filehandle($self->{fh});
+
+  return $self->{description} || [];
+}
